index.html

```diff
@@ -0,0 +1,300 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="App.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap" rel="stylesheet">
+    
+    <title>portfolio</title>
+</head>
+<body class >
+
+  <section  class="popup-container" >
+    <div class="pop" >
+       <!-- <div class="pop-window">
+        <div class="close-btn"><i class="fa fa-times"></i></div>
+        <h2 class="card-heading">Tonic</h2>
+        <div class="profesions">
+        <ul class="prof">
+            <li class="item">CANOPY </li>
+            <li class="item-prof">&#9679; Back End Dev  &#9679;  </li>
+            <li class="item-prof">2015</li>
+        </ul>
+    
+        </div>
+        <img src="./img/project-img/d-project-1.png" alt="first-project">
+        <span class="data-pop">
+          <p class="decription">Lorem Ipsum is simply dummy text of the printing and typesetting
+          industry. Lorem Ipsum has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a galley of type and
+          scrambled it to make a type specimen book. It has survived not only
+          five centuries, but also the leap into electronic typesetting,
+          remaining essent</p>
+          <div class="lon-p">
+            <ul class="lang-tag">
+              <li class="lang-list">HTML</li>
+              <li class="lang-list">CSS</li>
+              <li class="lang-list">JavaScript</li>
+              
+            </ul>
+            <a href="#">
+              <button class="see-btn">see live <img src="./img/social-media-icons/live.png" alt="live">
+              </button></a>
+            <a href="#"><button class="see-btn">see sorce <img src="./img/social-media-icons/github.png" alt="see more">
+            </button></a>
+        </span> 
+      </div>  -->
```

Comment:
- [ ] Kindly ensure that you remove all commented-out/ unused/ unwanted pieces of code from your code base. It makes your codebase look messy and cluttered. Having commented out code also violates the [HTML and CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).